include/openmc/distribution.h

```diff
@@ -23,7 +23,7 @@ class Distribution {
 public:
   virtual ~Distribution() = default;
   virtual double sample(uint64_t* seed) const = 0;
-
+  virtual double get_pdf(double x) const = 0;
```

Comment:
All this density function infrastructure is very nice to have, and it could be used for other applications outside point estimates.